src/datasources/__tests__/graphDb.test.js

```diff
@@ -145,7 +146,7 @@ describe('graphDb', () => {
             'CMR-Request-Id': 'abcd-1234-efgh-5678'
           })
           .post(
-            /graphdb/,
+            () => true,
```

Comment:
Is this like a wildcard? Should it look at `/example`?

Reply:
The `nock` call above matches the hostname of example.com. the `.post` matches the path, but there is no path for this new endpoint. This code is a workaround I found that will match every path.